WEB-INF/classes/gov/noaa/pfel/erddap/handlers/AxisVariableHandler.java

```diff
@@ -0,0 +1,86 @@
+package gov.noaa.pfel.erddap.handlers;
+
+import com.cohort.array.PAType;
+import com.cohort.array.PrimitiveArray;
+import com.cohort.util.String2;
+import org.xml.sax.Attributes;
+import org.xml.sax.SAXException;
+
+import java.util.ArrayList;
+
+public class AxisVariableHandler extends State {
+    private StringBuilder content = new StringBuilder();
+    private ArrayList tAxisVariables;
+    private String tSourceName = null, tDestinationName = null;
+    private com.cohort.array.Attributes tAttributes = null;
```

Comment:
This looks like you never assign to it.

Reply:
Again similar to the above its passed as an argument to the constructor on line 29,
`State state = new AddAttributesHandler(saxHandler, tAttributes, this);`